sample/command.conf

```diff
@@ -10,5 +10,8 @@
   @type groonga
   protocol command
   database /tmp/groonga/db
+
+  <buffer>
   flush_interval 1
```

Comment:
Could you add an indentation like as https://github.com/groonga/fluent-plugin-groonga/pull/17/commits/beaa8634b8438eb2266fcd14fba11a6071957fbd?